cmd/goreleaser/internal/configure.go

```diff
@@ -384,6 +400,7 @@ type distribution struct {
 	dockerSigns             []config.Sign
 	sboms                   []config.SBOM
 	checksum                config.Checksum
+	enableCgo               bool
```

Comment:
Can we move this `enableCgo` changes to a separate PR that we can land more quickly?

Reply:
Sure: https://github.com/open-telemetry/opentelemetry-collector-releases/pull/913